requirements-dev.txt

```
@@ -8,4 +8,4 @@ flake8-requirements==2.2.1
 mypy==1.15.0; implementation_name=="cpython"
 types-redis==4.6.0.20241004
 types-ujson==5.10.0.20250326
-aiocache-dynamodb==0.0.3 # used for docs
+aiocache-dynamodb==1.0.2 # used for docs
```

```suggestion
aiocache-dynamodb==1.0.2 # used for documentation
```
dummy change to trigger the action once more